Dumping_script.gm9

```diff
@@ -0,0 +1,309 @@
+# author: zeta , jisagi & Glazed_Belmont 
+
+@start
+set DISPLAY_MODE "PKMN Randomizer script\nTYPE: ______"
+
+labelsel -o -s "Is your game a cartridge or digital?" TYPE_*

```

Comment:
not a fan of this wording, maybe go with something like

"Are you dumping a cartridge or digital install?"